src/Payments/PaymentRequestButtonElement.res

```diff
@@ -94,5 +118,11 @@ let make = (~sessions, ~walletOptions, ~list: PaymentMethodsRecord.list) => {
     })
     ->React.array}
     <Surcharge list paymentMethod="wallet" paymentMethodType="google_pay" isForWallets=true />
+    <RenderIf
```

Comment:
Could you please move this renderIf inside the component itself

Reply:
I haven't done that because then I have to pass these values to that component unnecessarily. As these values are not getting used over there

Reply:
Updated